src/TreeMap/App.svelte

```diff
@@ -233,9 +315,9 @@
 </div>
 
 <style>
-    @import "https://api.mapbox.com/mapbox-gl-js/v1.2.0/mapbox-gl.css";
+    /* @import "https://api.mapbox.com/mapbox-gl-js/v1.2.0/mapbox-gl.css"; */
     @import "https://fonts.googleapis.com/css2?family=Raleway:wght@400;700&display=swap";
-
+    @import "map.css";
```

Comment:
Testing with a local HTML file, this is also tried to be loaded from local instead from the PFTP website:
<img width="823" alt="Bildschirmfoto 2021-01-28 um 15 32 31" src="https://user-images.githubusercontent.com/1532418/106152708-5994e280-61b0-11eb-83d5-61d3f72b4b93.png">